operators/src/util/raster_stream_to_geotiff.rs

```diff
@@ -118,9 +127,47 @@ where
                 }
 
                 // TODO: more descriptive error. This error occured when a file could not be created...
-                let dataset_writer = dataset_writer?;
+                let mut dataset_writer = dataset_writer?;
                 let tile = tile?;
 
+                let current_interval = tile.time;
+                let previous_interval = dataset_writer.time_intervals.last();
+
+                if previous_interval.is_none()
+                    || matches!(previous_interval, Some(x) if *x != current_interval)
+                {
+                    if previous_interval.is_some() {
+                        dataset_writer.finish()?;
+                    }
+                    dataset_writer.time_intervals.push(current_interval);
+                    let mut next_file_path: String =
+                        dataset_writer.placeholder_path.to_string_lossy().into();
+
+                    //TODO: Consider making time_string format and place_holder configurable
+                    let time_string = current_interval
+                        .start()
+                        .as_date_time()
+                        .ok_or(Error::TimeInstanceNotDisplayable)?
+                        .format(&DateTimeParseFormat::custom("%Y-%m-%d".to_string()));
```

Comment:
I guess without configuration is fine.
But ISO timestamps with date and time would cover most use cases, right?

Reply:
I did some refactoring for this to reduce lines of code for the method. 358486707b7e9cd6b2d3017c9e19b4c69df97de1
I added the time, but : is problematic in some OS, isn't it? So I used "-" for times as well. But you probably have more experience with weird timestamps in filenames and what to do there. 42b7121d68de34d829c79c9d4d91d0f82f52a17c